.buildkite/pipeline.yml

```diff
@@ -0,0 +1,55 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/bash-cache#add/cocoapods-support: ~
+  # Common environment values to use with the `env` key.
+  - &common_env
+    IMAGE_ID: xcode-12.5.1
+
+# This is the default pipeline – it will build and test the app
+steps:
+  #################
+  # Build and Test
+  #################
+  - label: "🧪 Build and Test"
+    key: "test"
+    command: .buildkite/build-and-test.sh
+    env: *common_env
+    plugins: *common_plugins
+
+  #################
+  # Validate Podspec
+  #################
+  - label: "🔬 Validating Podspec"
+    key: "validate"
+    command: .buildkite/validate-podspec.sh
+    env: *common_env
+    plugins: *common_plugins
+
+  #################
+  # Lint
+  #################
+  - label: "🧹 Lint"
+    command: .buildkite/lint.sh
+    plugins:
+      - automattic/bash-cache#add/cocoapods-support: ~
```

Comment:
```suggestion
    plugins:
      - *common_plugins
```

Reply:
I tried this (and a few other approaches) and couldn't get it to work – Buildkite is mad about this one too: https://buildkite.com/wordpress-mobile/wordpresskit-ios/builds/26#2bbf8117-25b2-4c1d-88b1-c7b0c8967cc1